frontend/src/pages/IstioConfigNew/GatewayForm/ListenerList.tsx

```diff
@@ -55,11 +55,12 @@ const columns: ThProps[] = [
   }
 ];
 
-export const addSelectorLabels = (value: string) => {
+export const addSelectorLabels = (value: string): [boolean, {}] => {
   if (value.length === 0) {
-    return;
+    return [true, {}];
```

Comment:
```suggestion
    return [false, {}];
```
Now that the label input is only enabled for `Selector` option, empty value should be invalid.